torchrl/collectors/vllm_weight_update.py

```diff
@@ -0,0 +1,181 @@
+import torch
+import threading
+
+from torchrl.collectors.weight_update import RemoteWeightUpdaterBase
+from torchrl.collectors.weight_update import LocalWeightUpdaterBase
+
+
+VLLM_ERR = None
+try:
```

Comment:
import `find_spec` might be preferred: https://docs.python.org/3/library/importlib.html#importlib.util.find_spec as it doesn't do the actual import until needed.

Reply:
I agree, that's the proper way to do it